packages/form-js-viewer/src/Form.js

```diff
@@ -173,12 +173,12 @@ export default class Form {
 
       const fieldErrors = validator.validateField(field, value);
 
-      return set(errors, path, fieldErrors.length ? fieldErrors : undefined);
+      return set(errors, [ pathStringify(path) ], fieldErrors.length ? fieldErrors : undefined);

```

Comment:
How is this related to the validation test?

Reply:
The reason using `path` instead of `[ pathStringify(path) ]` works is that we don't support nesting at the moment. Once we add support for nesting it would break as it would create nested objects instead of something like `documents.0.author: [ ... ]` as defined in the `@typedef` of `Errors`.